internal/runner/discover.go

```diff
@@ -12,7 +13,20 @@ type DiscoveryPattern struct {
 	ExcludePattern string
 }
 
-func discoverTestFiles(pattern DiscoveryPattern) ([]string, error) {
+func discoverTestFiles(defaultIncludePattern string) ([]string, error) {
+	includePattern := os.Getenv("BUILDKITE_SPLITTER_TEST_FILE_PATTERN")
```

Comment:
I have a refactor idea related to this part which I have captured in this ticket
https://linear.app/buildkite/issue/TAT-229/refactor-the-fetching-of-includeexclude-pattern-from-env-variables